src/neo4j_genai/schema.py

```diff
@@ -0,0 +1,136 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from typing import Any
+
+import neo4j
+
+
+BASE_ENTITY_LABEL = "__Entity__"
+EXCLUDED_LABELS = ["_Bloom_Perspective_", "_Bloom_Scene_"]
+EXCLUDED_RELS = ["_Bloom_HAS_SCENE_"]
+
+NODE_PROPERTIES_QUERY = """
+CALL apoc.meta.data()
+YIELD label, other, elementType, type, property
+WHERE NOT type = "RELATIONSHIP" AND elementType = "node" 
+    AND NOT label IN $EXCLUDED_LABELS
+WITH label AS nodeLabels, collect({property:property, type:type}) AS properties
+RETURN {labels: nodeLabels, properties: properties} AS output
+"""
+
+REL_PROPERTIES_QUERY = """
+CALL apoc.meta.data()
+YIELD label, other, elementType, type, property
+WHERE NOT type = "RELATIONSHIP" AND elementType = "relationship"
+    AND NOT label in $EXCLUDED_LABELS
+WITH label AS nodeLabels, collect({property:property, type:type}) AS properties
+RETURN {type: nodeLabels, properties: properties} AS output
+"""
+
+REL_QUERY = """
+CALL apoc.meta.data()
+YIELD label, other, elementType, type, property
+WHERE type = "RELATIONSHIP" AND elementType = "node"
+UNWIND other AS other_node
+WITH * WHERE NOT label IN $EXCLUDED_LABELS
+    AND NOT other_node IN $EXCLUDED_LABELS
+RETURN {start: label, type: property, end: toString(other_node)} AS output
+"""
+
+
+def _query(driver: neo4j.Driver, query: str, params: dict = {}) -> list[dict[str, Any]]:
```

Comment:
```suggestion
def _query_database(driver: neo4j.Driver, query: str, params: dict = {}) -> list[dict[str, Any]]:
```

Reply:
I don't know about this one. The entire package is about neo4j, so it kind of goes without saying that the query is to a database. Same as I think naming a module `neo4j_queries` is too verbose.

Reply:
For this package "querying" also means something else for the `query_text` and `query_vector` in `.search()` of the retriever classes so I think it's better to make this distinction.

The choice to name the module `neo4j_queries` is intentional as we will integrate with an external db like Weaviate

Reply:
Ok, makes sense! 👍

Reply:
[8fe6c5e](https://github.com/neo4j/neo4j-genai-python/pull/32/commits/8fe6c5e229fcfe150675639d7e89b71114a43033)